pkg/networkservice/mechanisms/kernel/kernelvethpair/client.go

```diff
@@ -60,23 +56,39 @@ func (k *kernelVethPairClient) Request(ctx context.Context, request *networkserv
 		Type: kernel.MECHANISM,
 	}
 	request.MechanismPreferences = append(request.MechanismPreferences, preferredMechanism)
-	conn, err := next.Client(ctx).Request(ctx, request, opts...)
+	var p peer.Peer
+	conn, err := next.Client(ctx).Request(ctx, request, append(opts, grpc.Peer(&p))...)
```

Comment:
Why we need a peer.Peer here? I do not see it is used.

Reply:
You are correct, that's an artifact of a previous approach that should have been trimmed from the code.  Removed.